Remote indexes (#169)
soundata/soundata: docs/source/contributing.rst

    Note that in this examples we group ``audio_voice1`` and ``audio_voice2`` in a single clip because the annotation ``voice-f0`` annotation corresponds to their mixture. In contrast, the annotation ``voice-f0`` is extracted from the multiclip mix and it is stored in the ``multiclips`` group. The multiclip ``multiclip1`` has an additional clip ``multiclip1-mix.wav`` which may be the master clip, the final mix, the recording of ``multiclip1`` with another microphone.

.. _upload_index:

3. Uploading the index to an online repository
----------------------------------------------

To prevent an the exponential growth in size of the library, we are storing all indexes in an online repository, while removing these from the repository itself.
For that task, we use Zenodo.
To use a dataloader, users may retrieve the index running the ``dataset.download()`` function that is also used to download the dataset.

> To download only the index, you may run ``.download(["index"])``. The index will be automatically downloaded and stored in the expected folder in Soundata.

Reviewer: I think that now the message should be improved. Can you please check @tanmayy24?

> To download only the index, you may run ``.download(["index"])``. The index will be automatically downloaded and stored in the expected folder in Soundata.

Reviewer: indeed. Fixed now :) Maybe it's a little bit overkill to use `if` `elif` `else` but that's the easiest way I found to log more meaningful messages here.

    Note that in this examples we group ``audio_voice1`` and ``audio_voice2`` in a single clip because the annotation ``voice-f0`` annotation corresponds to their mixture. In contrast, the annotation ``voice-f0`` is extracted from the multiclip mix and it is stored in the ``multiclips`` group. The multiclip ``multiclip1`` has an additional clip ``multiclip1-mix.wav`` which may be the master clip, the final mix, the recording of ``multiclip1`` with another microphone.

.. _upload_index:

3. Uploading the index to an online repository
----------------------------------------------

> To prevent an the exponential growth in size of the library, we are storing all indexes in an online repository, while removing these from the repository itself.
> For that task, we use Zenodo.

Reviewer 2: This sentence seems to make sense only for already existing indices. I guess new ones are never added to the repo in the first place:

```suggestion
We store all indexes in an online repository on Zenodo.
```

Reviewer: hey @hagenw, done! Thanks for the suggestion.
